src/cli_config.rs

```diff
@@ -113,6 +126,13 @@ pub fn cli_parse() -> CliConfig {
         .author(crate_authors!("\n"))
         .about(crate_description!())
         .setting(AppSettings::ColoredHelp)
+        .setting(AppSettings::SubcommandsNegateReqs)
+        .subcommand(SubCommand::with_name("config")
+            .arg(Arg::with_name("cfg-file")
+                .index(1)
+                .help("Load settings from saved config file. \
+                 Config file format is YAML")
```

Comment:
It seems it should be padded with 4 more spaces?

Reply:
done